views/js/runner/plugins/content/modalFeedback/modalFeedback.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016  (original work) Open Assessment Technologies SA;
+ *
+ * @author Alexander Zagovorichev <zagovorichev@1pt.com>
+ */
+
+/**
+ * @see http://www.imsglobal.org/question/qtiv2p1/imsqti_implv2p1.html#section10008 modalFeedback
+ */
+
+define([
+    'jquery',
+    'lodash',
+    'module',
+    'taoTests/runner/plugin',
+    'taoQtiTest/runner/plugins/content/dialog/itemInlineMessage',
+    'taoQtiTest/runner/plugins/content/dialog/itemAlertMessage',
+    'ui/autoscroll'
+], function ($, _, module, pluginFactory, inlineMessage, alertMessage, autoscroll) {
+    'use strict';
+
+    /**
+     * Modal or inline type of the messages
+     */
+    var inlineMode;
+
+    /**
+     * Form of the feedback
+     * by default dialog (modal) form
+     */
+    var messagePlugin;
+
+    /**
+     * All feedback messages
+     */
+    var renderedFeedbacks;
+
+    /**
+     * modalFeedback was resolved and all components were destroyed
+     */
+    var isDestroyed;
```

Comment:
It isn't works for me, example: modalFeedback creates 2 inlineMessages with only 1 ok button, when I press it should be runned modalFeedback.destroy(), but only once
step by step:
1. click on the OK button
https://github.com/oat-sa/extension-tao-testqti/blob/63529ce60329271550a69b6b57e8dc9a8b174390/views/js/runner/plugins/content/dialog/itemInlineMessage.js#L87-L87
2. for inline mode we have only one button, so needs to be run modalFeedback.destroy()
https://github.com/oat-sa/extension-tao-testqti/blob/39d77297ccee69eb09041ae9fc47792ccac6617e/views/js/runner/plugins/content/modalFeedback/modalFeedback.js#L154-L154
3. Destroy each of the feedback will halt his destroy (which again run click on the ok button for current feedback)